client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -11,28 +12,64 @@ import Card from 'components/card';
 import SectionHeader from 'components/section-header';
 import { getSelectedDomain } from 'lib/domains';
 import Button from 'components/button';
-import { requestTransferCode, enableDomainLocking } from 'lib/upgrades/actions';
+import { requestTransferCode, cancelTransferRequest } from 'lib/upgrades/actions';
 import notices from 'notices';
 import { displayRequestTransferCodeResponseNotice } from './shared';
 import support from 'lib/url/support';
 
 class Unlocked extends React.Component {
-	state = {
-		submitting: false
-	};
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			submitting: false,
+			// If the domain has any protection, we will must have sent the
+			// request to get to this page.
```

Comment:
Yeah, good call. I've taken your clearer verbiage, and added a mention of transfer-out/index too.